src/Menus/menu.json

```diff
@@ -118,9 +118,6 @@
                             "title": "WebStudio",
                             "url": "module=MyRadio,action=webstudio",
                             "description": "Access WebStudio. Plan, create, practise and broadcast your shows!"
-                        },
-                        {
-                            "template": "MyRadio/showPlanner.twig"
```

Comment:
Should we delete this template, so we don’t have loose files lying around?

Reply:
There's a load of dead showplanner templates etc that could be chucked. Back from the days when it was attempted to control BAPS2

Reply:
Eh, I'll file a separate issue for that to not block this.